packages/devextreme-angular/tests/src/ui/list.spec.ts

```diff
@@ -30,7 +29,7 @@ class TestContainerComponent implements AfterViewChecked {
     emptyItems = undefined;
     items = [1];
     complexItems = [{ text: 'Item 1' }];
-    emptyDataSource = { items: [] };
+    emptyDataSource: any = { items: [] };
```

Comment:
is it possible replace any to Record<string, unknown> ? (and in other places too)